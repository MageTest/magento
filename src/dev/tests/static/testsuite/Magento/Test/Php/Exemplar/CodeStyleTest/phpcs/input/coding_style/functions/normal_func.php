<?php
/**
 * Some function.
 * With long description.
 *
 * @param string|null $inParam
 * @return string
 */
function someFunc($inParam)
{
    if ($inParam === null) {
        $inParam = 'Stranger';
    }
    return 'Hello, ' . $inParam . '!';
}

/**
 * Another function with lot of parameters.
 * With long description.
 *
 * @param string|null $someLongParam
 * @param bool $anotherLongParam
 * @param int $moreEvenLongerParamForAllThatRoutineStuff
 * @param float $andThereGoesOneParameter
 * @return string
 */
function anotherFunc($someLongParam, $anotherLongParam, $moreEvenLongerParamForAllThatRoutineStuff,
    $andThereGoesOneParameter
) {
    if ($someLongParam === null) {
        $inParam = 'Stranger';
    }
    return 'Hello, ' . $someLongParam . '!';
}
