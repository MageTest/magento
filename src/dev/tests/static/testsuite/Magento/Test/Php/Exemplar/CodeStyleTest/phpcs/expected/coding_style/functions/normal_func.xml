<?xml version="1.0" encoding="UTF-8"?>
<config>
    <total errors="0"/>
</config>
